elixir/apps/domain/lib/domain.ex

```diff
@@ -34,6 +34,20 @@ defmodule Domain do
     end
   end
 
+  def html do
+    quote do
+      # Include general helpers for rendering HTML
+      unquote(html_helpers())
```

Comment:
We don't need to add macros here, doing `use Domain, :html` would be the same as `import Phoenix.HTML`, we should just use the later one in a few place where it's needed.

Reply:
I realized after following your suggestion that I didn't even need `import Phoenix.HTML`, so I just got rid of all of this and the places I was doing `use Domain, :html`